.github/workflows/auto-label-pr.yml

```
@@ -0,0 +1,77 @@
+          
+          This PR has been automatically labeled with \`${APPLIED_LABEL}\` based on linked issues in the Development section.
+          
+          **Release Type**: ${RELEASE_TYPE^} release
+          **How it works**:
+          1. ✅ Check Development section for linked issues
+          2. ✅ Extract release labels from linked issues
+          3. ✅ Apply highest priority label to PR (major > minor > patch)
+          4. ✅ Default to \`release:minor\` if no labels found
+          
+          This ensures consistent release labeling and enables automated release processes.
+EOF
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 76: could not find expected ':'

(syntax-check)